templates/site/node.html

```diff
@@ -3,9 +3,10 @@
 
 {% block content %}
     {% include "partials/banner.html" %}
-    <div class="container mt-3">
-        <div class="row g-0 border rounded d-flex flex-column position-static overflow-hidden flex-md-row mb-4 shadow-sm h-md-250 position-relative info-background">
-            <div class="col-12 col-sm-8 px-2">
+
+    {% with %}
```

Comment:
You can get rid of the with/endwith here now that you're not defining a content variable

Reply:
Removed the with/endwith